e2e/specs/components_iframe.spec.ts

```diff
@@ -37,4 +37,10 @@ describe("components.iframe", () => {
   it("sets scrolling correctly", () => {
     cy.get("iframe").should("have.attr", "scrolling", "auto");
   });
+
+  it("allows fullscreen content", () => {
```

Comment:
Is this overtesting? Especially since the other iframe allow policies don't have their own check here, and we already verify this behavior in unit tests... I'd have a (very weak) preference to just not include this at all.